firmware/config/engines/custom_engine.cpp

```diff
@@ -344,9 +344,6 @@ void setTle8888TestConfiguration() {
 	// IN2 PF14
 	// SF  PF11
 #if defined(STM32_HAS_GPIOF) && STM32_HAS_GPIOF
-#if EFI_FSIO
-	// todo lua setFsio(14, GPIOF_13, "1");
```

Comment:
i wonder what is that line about. today I am not excited about deciding if it's needed or not

Reply:
well:
- it's in a configuration called `test`
- that line hasn't been touched in 6 months, and nothing has blown up